CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - DeploymentLauncher can parse a .pb.json launch configuration.
 - DeploymentLauncher can launch a Simulated Player deployment independently from the target deployment.
 Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-deployment-name> <sim-deployment-name> <sim-deployment-json> <sim-deployment-region> <num-sim-players> <auto-connect>`
+- Clients will now validate schema against the server if the spatial runtime setting `Enable Schema Validation On Join` is enabled. 
```

Comment:
Ping docs team about this too since they are making a new UI page.

Reply:
They are already on this, I believe there are some draft docs already

Reply:
Will do!